src/pages/main_layout.cr

```diff
@@ -110,8 +110,8 @@ abstract class MainLayout
 
     ## What does Lucky look like?
     
-    In Lucky we generate separate action classes with names that describe the route.
-    The resulting file maps routing path definitions to a response block.
+    In Lucky we generate **action classes** with names that describe the route.
+    The classes then maps a routing path definition to a response block.
```

Comment:
I believe `maps` -> `map` would read better

Reply:
Yep.
Added some more context to the first sentence as well.